activemodel/lib/active_model/attributes.rb

```diff
@@ -117,6 +134,26 @@ def attribute_names
       @attributes.keys
     end
 
+    # Returns a string representation including attributes
+    #
+    #   class Person
+    #     include ActiveModel::Attributes
+    #
+    #     attribute :name, :string
+    #     attribute :age, :integer
+    #   end
+    #
+    #  person = Person.new(name: "David", age: 41)
+    #  person.inspect
+    #  # => #<Person name: "David", age: 41>
+    def inspect
+      inspection = attributes.map do |name, value|
+        "#{name}: #{format_for_inspect(name, value)}"
+      end.join(", ")
+
+      "#<#{self.class} #{inspection}>"
+    end
+
```

Comment:
This method seems like the only one that really changed. Not sure if it’s better have seperate implementations or if ActiveRecord could call super, in some cases, to call this method.